src/main.rs

```diff
@@ -919,28 +922,52 @@ mod tests {
         );
     }
 
+    #[test]
+    fn test_function_call() {
+        parse_str(
+            r#"
+                fn test() {}
+                test()
```

Comment:
I'm not yet sure if we want to support calling function from not within a code body because other than js/ts i'm not sure of any language that support this.

Reply:
Ah, yes. I am still thinking from a JavaScript dev view as I have not worked with C/C++ and others much. I'll remove this for now.